init.c

```diff
@@ -490,7 +509,7 @@ int main(int argc, char *argv[])
 		goto loop;
 	}
 
-	mount_cgroups();
+	other_mounts();
```

Comment:
i dont like the name other_mounts.... feels like we havent thought enough what these mounts actually are :)

Reply:
not sure about how to rename it. Maybe we can divide the function and call them mount_disks (for /writable and /exports) and mount_home (/root and /run?)